models/question.ts

```diff
@@ -0,0 +1,11 @@
+const { Pool } = require('pg');
+
+const pool = new Pool();
+const Logger = require('../models/logger');
+
+
+module.exports.getQuestions = function (questionID, callback) {
+    const sql = 'SELECT * FROM question, answer WHERE question.questionid = answer.questionid AND question.questionID = $1';
```

Comment:
The naming convention for Postgres is to have the tables as plural. So questions, answers. But fontys taught us differently (as we learned mssql) so this is fine for now.

Reply:
Oh never knew that, at fontys they always told me to use singular.